baseplate/clients/redis.py

```diff
@@ -194,20 +194,19 @@ def execute_command(self, *args: Any, **kwargs: Any) -> Any:
         command = args[0]
         trace_name = f"{self.context_name}.{command}"
 
-        with self.server_span.make_child(trace_name):
+        labels = {
+            f"{PROM_LABELS_PREFIX}_command": command,
+            f"{PROM_LABELS_PREFIX}_client_name": self.connection_pool.connection_kwargs.get(
+                "client_name", ""
+            ),
+            f"{PROM_LABELS_PREFIX}_database": self.connection_pool.connection_kwargs.get("db", ""),
+            f"{PROM_LABELS_PREFIX}_type": "standalone",
+        }
+        with self.server_span.make_child(trace_name), ACTIVE_REQUESTS.labels(
+            **labels
+        ).track_inprogress():
```

Comment:
This is a bit hard to read. How does this look:
```suggestion
        with (
            self.server_span.make_child(trace_name), 
            ACTIVE_REQUESTS.labels(**labels).track_inprogress(),
        ):
```